src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -118,6 +118,14 @@ export type MetricActionTemplate = {
    * Integration id for this `type`, should be passed to backend as `integrationId` when creating an action
    */
   integrationId: number;
+  /**
+   * The input type on the front end, e.g. text for Slack, select for PagerDuty
+   */
+  inputType?: string;
```

Comment:
Enum for this like TargetType ActionType would be nice or maybe just 'select' | 'text' 🤷  would work too